guides/index.md

```diff
@@ -1,5 +1,5 @@
-The Ember CLI (command line interface) is the official way to create, build, test, and serve the files that make up an Ember.js app or addon.
-Many things have to happen before any web app is ready for the browser, and the Ember CLI helps you get there with zero configuration.
+The Ember CLI (command line interface) is the official way to create, build, test, and serve the files that make up an Ember app or addon.
+Many things have to happen before a web app is ready for the browser. The Ember CLI helps you get there with zero configuration.
```

Comment:
I think generally "the Ember CLI" is a bit odd. I'd personally say:

```suggestion
Many things have to happen before a web app is ready for the browser. Ember CLI helps you get there with zero configuration.
```

Reply:
Yep, I think removing the article sounds better. Thanks for the suggestion!